src/pseudopeople/default_configuration.yaml

```diff
@@ -160,9 +184,15 @@ taxes_w2_and_1099:
     mailing_address_street_number:
         missing_data:
             row_noise_level: 0.01
+        numeric_miswriting:
```

Comment:
I have this in my PR but you'd better add it here as well. This config is currently missing `mailing_address_po_box` for the tax forms.

```
mailing_address_po_box:
        missing_data:
            row_noise_level: 0.01
```